cpanfile

```diff
@@ -23,7 +23,7 @@ requires 'Selenium::Remote::WDKeys';
 requires 'Digest::file';
 requires 'YAML::Tiny';
 requires 'Test::Assert';
-
```

Comment:
If you keep this blank line (below the new line that you added) it will look nicer.

Reply:
ahm thanks! now I get why that line :)